bitbots_vision/src/bitbots_vision/vision_modules/candidate.py

```diff
@@ -12,135 +12,106 @@ def __init__(self, x1=0, y1=0, width=0, height=0, rating=None):
     def get_width(self):
         # type: () -> int
         """
-        returns...
```

Comment:
Even tho this is self-explanatory, I would also include some description like `Returns the width in <units> of the candidate.`.

Also in the cases below...

Reply:
Putting it in the Description and the `:returns:` part is a duplication. But we can indeed improve the `returns` part.